apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/GraphqlFragment.kt

```diff
@@ -1,14 +1,3 @@
 package com.apollographql.apollo.api
 
-import com.apollographql.apollo.api.internal.ResponseFieldMarshaller
-
-/**
- * Represents a GraphQL fragment
- */
-interface GraphqlFragment {
-
-  /**
-   * Returns marshaller [ResponseFieldMarshaller] to serialize fragment data
-   */
-  fun marshaller(): ResponseFieldMarshaller
-}
+interface GraphqlFragment
```

Comment:
nit: keep KDoc?

Reply:
It's not that helpful but it does make the file nicer :D ! Overall, I'd like to put a more descriptive KDoc but I'm not sure what to put there. It actually feels like this interface isn't really useful anymore...